### DISCUSSION TITLE
Automaticly change to another channel

### DISCUSSION THREAD

**Discussion by AlifPapp** [victornpb/undiscord#326] [3. Ideas] [answered]
Sometimes you want to delete all messages in a single guild, so it would be nice if you didn't have to keep manually changing the channels.

**Comment by ghost** [comment, marked as answer]
https://github.com/victornpb/undiscord/blob/master/help/channelId.md#entire-server

this feature already exists, see above

**Reply by victornpb** [reply]
Yes, you can leave the Channel ID empty.

NOTE: Only works on servers, for DMs you have to fill both boxes.

**Reply by AlifPapp** [reply]
> Yes, you can leave the Channel ID empty.
> 
> NOTE: Only works on servers, for DMs you have to fill both boxes.

thnx
